cpp/src/io/json/parser_features.cpp

```diff
@@ -88,7 +88,15 @@ struct empty_column_functor {
     std::unique_ptr<column> child = cudf::type_dispatcher(
       schema.child_types.at(child_name).type, *this, schema.child_types.at(child_name));
     auto offsets = make_empty_column(data_type(type_to_id<size_type>()));
-    return make_lists_column(0, std::move(offsets), std::move(child), 0, {}, stream, mr);
+    std::vector<std::unique_ptr<column>> child_columns;
+    child_columns.push_back(std::move(offsets));
+    child_columns.push_back(std::move(child));
+    return std::make_unique<column>(cudf::data_type{type_id::LIST},
+                                    0,
+                                    rmm::device_buffer{},
+                                    rmm::device_buffer{},
+                                    0,
+                                    std::move(child_columns));
```

Comment:
Why is this change needed?
Could this utility be used instead? https://github.com/rapidsai/cudf/blob/9cc907122077d18e5128e7da36685fdeb82fef41/cpp/include/cudf/lists/detail/lists_column_factories.hpp#L50-L52

Reply:
It works only for fixed width type child.

Reply:
This change is to avoid any `purge/has_nonempty_nulls` calls.